### PR TITLE
Update eco bridge adapter with the new bridge addresses for the ECO bridges on OP and OP Goerli

### PR DESCRIPTION
**Pull request by tremarkley**
In https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/387/files the addresses for the ECO bridges were updated. This change updates the ECO SDK bridge adapter to point to the new addresses.

### CODE REVIEW

**Review by roninjin10 — APPROVED** [review]

**Review by nickbalestra — APPROVED** [review]
Approving as it LGTM, But I would prefer for @roninjin10 to have a final pass and stamp of approval before landing this

**Review by zhwrd — APPROVED** [review]

**Pull request merged**
